.env.sample

```diff
@@ -5,4 +5,7 @@ EMAIL_FROM=
 JWT_SECRET=
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
-AWS_REGION=
\ No newline at end of file
+AWS_REGION=
+STRIPE_SECRET_KEY=
```

Comment:
Do we need a STRIPE_TEST_KEY as well?

Reply:
No, we don't need that. Stripe provides a production key and a test key so we'll be using the test key in our local `.env` file and the production key in the production `.env` file for the `STRIPE_SECRET_KEY` variable.